scipion/__main__.py

```diff
@@ -344,13 +344,19 @@ def main():
 
     elif mode == MODE_ENV:
         # Print all the environment variables needed to run scipion.
-        from pyworkflow.plugin import Plugin
+        from pyworkflow.utils import greenStr,yellowStr
 
         # Trigger plugin's variable definition
         pyworkflow.Config.getDomain().getPlugins()
-        VARS.update(pyworkflow.plugin.Plugin.getVars())
-        for key in sorted(VARS):
-            sys.stdout.write('export %s="%s"\n' % (key, VARS[key]))
+
+        for key, var in pyworkflow.VariablesRegistry.variables().items():
```

Comment:
Aha, that's why the output is not sorted by name!

Reply:
was it handy?